src/codeu/chat/ServerMain.java

```diff
@@ -95,10 +116,66 @@ private static void runServer(Uuid id,
                         new NoOpRelay() :
                         new RemoteRelay(relaySource);
 
-    final Server server = new Server(id, secret, relay);
+    //name of persistentLog file              
+    String persistentFileName = persistentPath + "//persistentLog.txt";
```

Comment:
This would be a good place to set the distinction, by using a name that would be more specific per port.